site/src/pages/UserSettingsPage/SSHKeysPage/SSHKeysPage.tsx

```diff
@@ -19,12 +20,13 @@ export const Language = {
     "You will need to replace the public SSH key on services you use it with, and you'll need to rebuild existing workspaces.",
   confirmLabel: "Confirm",
   cancelLabel: "Cancel",
+  errorRegenerateSSHKey: "Error on regenerate the SSH Key",
 }
 
 export const SSHKeysPage: React.FC = () => {
   const xServices = useContext(XServiceContext)
```

Comment:
I think you'll find that Storybook won't work because of the XServices here. My plan for cases like these was actually to break this up into two components, so that the rendering happened in a different one that we can Storybook. It's up to you whether to do that in this PR or write a chore ticket for us to get to it later.